nltk/test/unit/lm/test_preprocessing.py

```diff
@@ -13,16 +13,16 @@ class TestPreprocessing(unittest.TestCase):
     def test_padded_everygram_pipeline(self):
         expected_train = [
             [
-                ("<s>",),
```

Comment:
I've been thinking about these tests a bit. We don't really care about the order of the items in the sequence right? The important thing is that we "see" all the different ngram orders. Let's future-proof our tests by using sets instead of lists wherever possible. For instance, doctests still care about the order of items because they check the raw output of `print`, but all of our unittests would really benefit from the conversion.

Reply:
I think using sets here makes sense!

Reply:
coolio, thanks for taking care of this!